cmd/operator/internal/controller/kubearchiveconfig_controller.go

```diff
@@ -39,9 +39,12 @@ const (
 	ApiServerSourceLabelValue = "true"
 )
 
-var k9eNs = os.Getenv("KUBEARCHIVE_NAMESPACE")
-var a13eName = k9eNs + "-a13e"
-var k9eSinkName = k9eNs + "-sink"
+var (
+	k9eNs         = os.Getenv("KUBEARCHIVE_NAMESPACE")
+	a13eName      = k9eNs + "-a13e"
+	k9eSinkName   = k9eNs + "-sink"
+	k9eBrokerName = k9eNs + "-broker"
```

Comment:
I can fix this issue here as part of this PR since I don't think it would be that difficult. We just need to set a new env var for the operator. Something like:

```
- name: KUBEARCHIVE_PREFIX
  value: {{ .Release.Name }}
```

Maybe we want that do be done separately?

Reply:
Lets fix it in another PR, this one is already good as is, even if it will fail under some circumstances (changing the namespace).